morpheus/llm/services/nemo_llm_service.py

```diff
@@ -214,11 +217,12 @@ def __init__(self, *, api_key: str = None, org_id: str = None, retry_count=5) ->
         super().__init__()
         api_key = api_key if api_key is not None else os.environ.get("NGC_API_KEY", None)
         org_id = org_id if org_id is not None else os.environ.get("NGC_ORG_ID", None)
+        base_url = base_url if base_url is not None else os.environ.get("NGC_API_BASE", None)
```

Comment:
I think we want the environment to overwrite the variable set in the code. So this should be in the form: `os.environ.get("NGC_API_BASE", base_url)`